src/chatkit.ts

```diff
@@ -277,6 +349,14 @@ export default class Chatkit {
     }).then(() => {})
   }
 
+  getUser(options: GetUserOptions): Promise<any> {
+    return this.apiInstance.request({
+      method: 'GET',
+      path: `/users/${encodeURIComponent(options.id)}`,
+      jwt: this.getServerToken(),
+    }).then(({ body }) => JSON.parse(body))
+  }
+
   getUsers(): Promise<any> {
```

Comment:
This should support `fromTimestamp` (translated to `from_ts` in the query string) in its options I believe

Reply:
Hmmm yeah, well spotted. Will add it now.